.circleci/config.yml

```diff
@@ -185,7 +185,9 @@ jobs:
     working_directory: ~/identity-idp
 
     steps:
-      - browser-tools/install-browser-tools
+      - browser-tools/install-browser-tools:
+          install-firefox: false
+          install-geckodriver: false
```

Comment:
do you think it would make sense to define our own step and reuse that in all these places instead of having to copy-paste the same stanza?

Reply:
Thanks, yep. Added in cba38c6